src/Adapter/Order/CommandHandler/AddCartRuleToOrderHandler.php

```diff
@@ -101,10 +103,6 @@ public function handle(AddCartRuleToOrderCommand $command): void
      */
     private function addCartRuleAndUpdateOrder(AddCartRuleToOrderCommand $command, Order $order): void
     {
-        $computingPrecision = new ComputingPrecision();
```

Comment:
I dont see why these 3 lines are removed 🤔

Reply:
Because they are actually useless 😅. 
At first I only removed `$precision` which was not used, then it highlighted the fact that the other two were also useless